packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx

```diff
@@ -0,0 +1,284 @@
+import Step from '@mdx/components/Step';
+import Tabs from '@mdx/components/Tabs';
+import TabItem from '@mdx/components/TabItem';
+
+<Step
+  title="Install SDK"
+  subtitle="Install Logto Android SDK with Gradle"
+  index={0}
+  activeIndex={props.activeStepIndex}
+  invalidIndex={props.invalidStepIndex}
+  onNext={() => props.onNext(1)}
+>
+
+## Step1: Install Logto Android SDK with Gradle
+
+### Prerequisite
+* Minimal Android SDK: Level 24 (TBD)
+
+<Tabs>
+
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
+dependencies {
+	implementation("io.logto.sdk:android:1.0.0")
+}
+```
+
+</TabItem>
+
+<TabItem value="groovy" label="Groovy">
+
+```groovy
+dependencies {
+	implementation 'io.logto.sdk:android:1.0.0'
+}
+```
+
+</TabItem>
+
+</Tabs>
+
+</Step>
+
+<Step
+  title="Configure"
+  subtitle="Configure your application and LogtoClient"
+  index={1}
+  activeIndex={props.activeStepIndex}
+  invalidIndex={props.invalidStepIndex}
+  onNext={() => props.onNext(2)}
+>
+
+## Step2: Configure
+
+### Configure Redirect URI
+
+The `Redirect URI` indicates the endpoint that the application used to receive authorization results, which is implemented by the Logto Android SDK internally.
+
+### Configure Redirect URI in Admin Console
+
+Add the following value to Redirect URIs on the application settings page.
+
+```bash
+$(LOGTO_REDIRECT_SCHEME)://$(YOUR_APP_PACKAGE)/callback
+```
+
+Notes: 
+- `LOGTO_REDIRECT_SCHEME` should be a custom scheme in the reverse domain format.
+- Replace `$(LOGTO_REDIRECT_SCHEME)` with your own settings.
+
+e.g. `io.logto.android://io.logto.sample/callback`
+
+### Configure Logto Android SDK
+
+<Tabs>
+
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
+import io.logto.sdk.android.LogtoClient
+import io.logto.sdk.android.type.LogtoConfig
+
+class MainActivity : AppCompatActivity() {
+		val logtoConfig = LogtoConfig(
+            endpoint = "<your-logto-endpoint>",
+            appId = "<your-app-id>",
+            scopes = null,
+            resources = null,
+            usingPersistStorage = true,
+        )
+
+		val logtoClient = LogtoClient(logtoConfig, application)
+}
```

Comment:
wrong ident

Reply:
Android mdx is a direct copy from `docs`, maybe @xiaoyijun can help double check this?